examples/script/zlib.yml

```diff
@@ -2,18 +2,28 @@ version: 0.0.1
 
 zlib_compression:
   type: script
+  env:
+    FILE: 1mb.txt
+    TARFILE: 1mb.tar.gz
   run: |
-    dd if=/dev/zero of=1mb.txt bs=1M count=1
-    tar cvzf 1mb.tar.gz 1mb.txt
-    ls -lh 1mb.tar.gz
-    gzip -l 1mb.tar.gz
+    dd if=/dev/zero of=$FILE bs=1M count=1
+    tar cvzf $TARFILE $FILE
```

Comment:
Is there a reason you don't quote bash variables or put them in `{}`? I thought that was considered best practice.

Reply:
that's a typo on my end i will use ``{}``